apps/builddao/widget/components/Modal.jsx

```diff
@@ -0,0 +1,76 @@
+const { theme } = VM.require("rambo-dev.near/widget/ThemeProvider");
```

Comment:
Can we not use the ThemeProvider please? We can explore this in the future.

Theming does not need to happen on a per-component basis. We will just define the Theme in the root.

Reply:
Totally